src/main/java/HTTP/HTTPBasicAuthClient.java

```diff
@@ -0,0 +1,26 @@
+package HTTP;
+
+import org.apache.http.HttpHost;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import static HTTP.HTTPVariables.*;
+
+public class HTTPBasicAuthClient extends HTTPClient {
```

Comment:
I don't see that reason why HTTPBasicAuthClient extend HTTPClient. I don't see anything common except name :)